Movie/Movie/DetailModule/View/ActorCollectionViewCell.swift

```diff
@@ -54,25 +50,20 @@ final class ActorCollectionViewCell: UICollectionViewCell {
 
     // MARK: - Public Metods
 
-    func configureCell(_ actror: Actor, networkService: NetworkServiceProtocol) {
-        actorNameLabel.text = actror.name
-        actorRoleLabel.text = actror.character
-        guard let profilePath = actror.profilePath
+    func configureCell(_ actor: Actor, detailPresenter: DetailPresenerProtocol) {
+        detailPresenter.realmService?.saveToRealm(object: actor)
```

Comment:
```suggestion
        detailPresenter.save(object: actor)
```
 а уже в презентере внутри save идет вызов реалмсервиса